beacon/validator/src/test/java/tech/pegasys/teku/validator/coordinator/AbstractBlockFactoryTest.java

```diff
@@ -291,6 +299,49 @@ protected SignedBeaconBlock blindSignedBeaconBlockIfUnblinded(
     return spec.blindSignedBeaconBlock(unblindedSignedBeaconBlock);
   }
 
+  protected BlockAndBlobs assertBlobSidecarsCreated(final boolean blinded, final Spec spec) {
```

Comment:
I don't like this a bit. It's not clear that block is generated inside and nothing is tested for block. It's ok when we want to test blobsidecars only but the class is `AbstractBlockFactoryTest`. And we cannot name it `assertBlockAndBlobSidecarsCreated` because we don't test anything for blocks here. Maybe we could split it to all block things in `prepareBlock` and `assertBlobSidecarsCreated`?

Or maybe just rename it to `createBlobSidecars`, and move this single assertion to every calling test, it will cover almost all of my comments

Reply:
Nice. Thanks for the comment. Went for the `produceBlobSidecars` route. Think it makes it cleaner. I agree.